### ISSUE TITLE
Can't configure SMTP settings

### ISSUE THREAD

**Issue by sameersbn** [bitnami/bitnami-docker-wordpress#30]
Container launch fails when SMTP settings are specified. I used the following docker-compose.yml file for the test:

```yaml
version: '2'
services:
  mariadb:
    image: 'bitnami/mariadb:latest'
    volumes:
      - 'mariadb_data:/bitnami/mariadb'
  application:
    image: 'bitnami/wordpress:latest'
    environment:
      - WORDPRESS_USERNAME=bobloblaw
      - WORDPRESS_PASSWORD=passw0rd
      - WORDPRESS_EMAIL=bobloblaw@thelawofficesofbobloblaw.com
      - WORDPRESS_FIRST_NAME=Bob
      - WORDPRESS_LAST_NAME=LobLaw
      - WORDPRESS_BLOG_NAME=Bob LobLaw's Law Blog
      - SMTP_HOST=smtp.gmail.com
      - SMTP_PORT=587
      - SMTP_USER=notifications@example.com
      - SMTP_PASSWORD=secretpassword
      - SMTP_USERNAME=noreply@example.com
      - SMTP_PROTOCOL=tls
    ports:
      - '10080:80'
      - '10443:443'
    volumes:
      - 'wordpress_data:/bitnami/wordpress'
      - 'apache_data:/bitnami/apache'
    depends_on:
      - mariadb
volumes:
  mariadb_data:
    driver: local
  wordpress_data:
    driver: local
  apache_data:
    driver: local
```

Output:

```bash
$ docker-compose up 
Creating volume "wordpress_apache_data" with local driver
Creating volume "wordpress_wordpress_data" with local driver
Creating volume "wordpress_mariadb_data" with local driver
Creating wordpress_mariadb_1
Creating wordpress_application_1
Attaching to wordpress_mariadb_1, wordpress_application_1
mariadb_1      | nami    INFO  Initializing mariadb
application_1  | nami    INFO  Initializing apache
application_1  | apache  INFO  ==> Patching httpoxy...
application_1  | nami    INFO  apache successfully initialized
application_1  | nami    INFO  Initializing php
application_1  | nami    INFO  php successfully initialized
application_1  | nami    INFO  Initializing wordpress
application_1  | mysqlCo INFO  Trying to connect to MySQL server
mariadb_1      | mariadb INFO  ==> Creating 'root' user with unrestricted access...
mariadb_1      | mariadb INFO  ==> Flushing privileges...
mariadb_1      | mariadb INFO  ==> Enabling remote connections...
mariadb_1      | mariadb INFO 
mariadb_1      | mariadb INFO  ########################################################################
mariadb_1      | mariadb INFO   Installation parameters for mariadb:
mariadb_1      | mariadb INFO     Root Password: Not set during installation
mariadb_1      | mariadb INFO     Password: Not set during installation
mariadb_1      | mariadb INFO   (Passwords are not shown for security reasons)
mariadb_1      | mariadb INFO  ########################################################################
mariadb_1      | mariadb INFO 
mariadb_1      | nami    INFO  mariadb successfully initialized
mariadb_1      | Starting application ...
mariadb_1      | 
mariadb_1      |   *** Welcome to the mariadb image ***
mariadb_1      |   *** Brought to you by Bitnami ***
mariadb_1      |   *** More information: https://github.com/bitnami/bitnami-docker-mariadb ***
mariadb_1      |   *** Issues: https://github.com/bitnami/bitnami-docker-mariadb/issues ***
mariadb_1      | 
mariadb_1      | 
mariadb_1      | *** Your container is up to date! ***
mariadb_1      | 
mariadb_1      | 2016-09-05 10:16:07 139638273099712 [Note] InnoDB: Highest supported file format is Barracuda.
mariadb_1      | 2016-09-05 10:16:07 139638273099712 [Note] InnoDB: 128 rollback segment(s) are active.
mariadb_1      | 2016-09-05 10:16:07 139638273099712 [Note] InnoDB: Waiting for purge to start
mariadb_1      | 2016-09-05 10:16:07 139638273099712 [Note] InnoDB:  Percona XtraDB (http://www.percona.com) 5.6.29-76.2 started; log sequence number 1616829
mariadb_1      | 2016-09-05 10:16:07 139637733435136 [Note] InnoDB: Dumping buffer pool(s) not yet started
mariadb_1      | 2016-09-05 10:16:07 139638273099712 [Note] Plugin 'FEEDBACK' is disabled.
mariadb_1      | 2016-09-05 10:16:08 139638273099712 [Note] Server socket created on IP: '0.0.0.0'.
mariadb_1      | 2016-09-05 10:16:08 139638273099712 [Warning] 'proxies_priv' entry '@% root@e7eda230581e' ignored in --skip-name-resolve mode.
mariadb_1      | 2016-09-05 10:16:08 139638273099712 [Note] /opt/bitnami/mariadb/sbin/mysqld: ready for connections.
mariadb_1      | Version: '10.1.14-MariaDB'  socket: '/opt/bitnami/mariadb/tmp/mysql.sock'  port: 3306  Source distribution
application_1  | mysqlCo INFO  Found MySQL server listening at mariadb:3306
application_1  | mysqlCo INFO  MySQL server listening and working at mariadb:3306
application_1  | Error executing 'postInstallation': You should specify username, password, host, port for SMTP configuration
wordpress_application_1 exited with code 1
```

**Comment by prydonius** [comment]
Did #33 fix this?

**Comment by juan131** [comment]
It's fixed in #33. Furthermore, I included a new check for the parameters. The next revision will have this issue fixed.

**Comment by sameersbn** [comment]
I can confirm that the issue has been resolved since release `4.6.1-r0`
